tests/storage/storage.go

```diff
@@ -700,7 +688,7 @@ var _ = SIGDescribe("Storage", func() {
 			})
 			It("VirtioFS, it should fail to start a VMI", func() {
 				tests.DisableFeatureGate(virtconfig.VirtIOFSGate)
-				vmi := tests.NewRandomVMIWithFSFromDataVolume("something")
+				vmi := libvmi.NewFedora(libvmi.WithFilesystemDV("disk1", "something"))
```

Comment:
it does not matter here, as any string would work, but might be misleading, as the first argument is a filesystem name not a disk name

Reply:
Changed it to `fs0`